src/runtime/runtime.js

```diff
@@ -27,12 +27,10 @@
   var $TypeError = TypeError;
   var $create = $Object.create;
   var $defineProperty = $Object.defineProperty;
-  var $freeze = $Object.freeze;
+  var {freeze} = $Object;
```

Comment:
I was thinking I would use destructuring to save some characters... not worth it. Changing back.